update-vanadium-patches.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+
+gpwd=$(pwd)
+readonly path="./vanadium_patches/"
+readonly url=https://raw.githubusercontent.com/GrapheneOS/Vanadium/main/patches
+patches=() # current patch names
+smallpatches=() # truncated version of the patches for comparison
+vpatches=() # vanadium patches
+svpatches=() # truncated version for comparisons
+
+# gets the current patches
+get_patches() {
+	cd $path
+	patches=(*.patch)
+	for ((i=0; i<${#patches[@]}; i++)); do
+		smallpatches[$i]="${patches[$i]:4}"
```

Comment:
on the off chance vanadium changes their format, can we make the script fail if any of the patches don't start with four integers?